grammar.js

```diff
@@ -723,6 +724,59 @@ module.exports = grammar({
       ),
     ),
 
+    comment_text: $ => $._literal_string,
```

Comment:
comment text is a reasonably special case imo but consider the common `alias($._literal_string, $.literal)` pattern here!

Reply:
Ah, I was already wondering how to get a literal string to be visible in the test cases, I hadn't figured out `alias` yet. Thanks!